packages/date-picker/src/DatePicker/components/Calendar/Calendar.tsx

```diff
@@ -79,13 +94,19 @@ export const Calendar: React.VFC<CalendarProps> = ({
         }
       >
         <DayPicker
-          selectedDays={value}
+          selectedDays={range ? selectedRange : value}
           initialMonth={value ? value : initialMonth}
           firstDayOfWeek={firstDayOfWeek}
           disabledDays={disabledDays}
           onDayClick={onDayChange}
           navbarElement={getNavbar}
           classNames={defaultCalendarClasses}
+          modifiers={
+            {
+              [calendarStyles.from]: modifiers?.from,
+              [calendarStyles.to]: modifiers?.to,
+            } as Modifiers
+          }
```

Comment:
Unfortunately recasting this object is the only way we've got this to work with Typescript. We need to pass the modifiers in like this to get the styled component class names as from/to aren't typed in the library.